docs/HACKING.md

```diff
@@ -55,7 +59,10 @@ Add the following lines to your shell startup script (`.bashrc`, `.zshrc`, etc.)
 
     $ mkvirtualenv -a $PROJECT_HOME/goldstone-server goldstone-server
 
-   Copy this [postactivate](https://gist.github.com/jxstanford/6ee6cc61143113776d0d#file-postactivate) script into your `$WORKON_HOME/goldstone-server/bin` folder, overwriting the original.
+   Copy this [postactivate](https://gist.github.com/jxstanford/6ee6cc61143113776d0d#file-postactivate) script into your `$WORKON_HOME/goldstone-server/bin` folder, overwriting the original. Then:
```

Comment:
The postactivate script points DJANGO_SETTINGS_MODULE at docker_dev.py.  

Docker_dev.py defines the database as being on host gsdb.

Running python interactively, I can't get to host gsdb:

<img width="1126" alt="screen shot 2015-09-23 at 3 21 22 pm" src="https://cloud.githubusercontent.com/assets/231148/10060643/28b7c6cc-6207-11e5-8f5d-c1f8f8266fd2.png">